ANONYMITY_NETWORKS.md

```diff
@@ -74,6 +79,36 @@ forwarded to `monerod` localhost port 30000.
 These addresses will be shared with outgoing peers, over the same network type,
 otherwise the peer will not be notified of the peer address by the proxy.
 
+### Wallet RPC
+
+An anonymity network can be configured to forward incoming connections to a
+`monerod` RPC port - which is independent from the configuration for incoming
+P2P anonymity connections. The anonymity network (tor, i2pd, kovri, etc.) is
```

Comment:
Minor request, can we change "(tor, i2pd, kovri, etc.)" to simply "tor/i2p"? This keeps implementation names out.

Reply:
Will update shortly.

Reply:
Thanks